### PR TITLE
Add GX_SetTevIndWarp Function

### PR DESCRIPTION
**Pull request by gamemasterplc**
The function provides a wrapper to GX_SetTevIndirect to warp indirect textures more easily. The function is also undocumented at the moment.

### CODE REVIEW

File: gc/ogc/gx.h [hunk 1]
```diff
@@ -3418,6 +3418,8 @@ void GX_SetTevIndBumpXYZ(u8 tevstage,u8 indstage,u8 mtx_sel);
  */
 void GX_SetTevIndTile(u8 tevstage,u8 indtexid,u16 tilesize_x,u16 tilesize_y,u16 tilespacing_x,u16 tilespacing_y,u8 indtexfmt,u8 indtexmtx,u8 bias_sel,u8 alpha_sel);
 
+void GX_SetTevIndWarp(u8 tevstage, u8 indtexid, u8 bias_flag, u8 replace_tex, u8 mtxid);
```
Comment (WinterMute) [inline gc/ogc/gx.h]: Could you add doxygen to explain what this function is for please?

**Comment by gamemasterplc** [comment]
Merge it @WinterMute

**Comment by WinterMute** [comment]
> Merge it @WinterMute

This is not appropriate use of the issue trackers. Please don't make demands of maintainers.

**Comment by gamemasterplc** [comment]
Would someone please merge this pull request for me because I can't merge it.

**Comment by WinterMute** [comment]
It's not appropriate to demand that PRs get merged, please exercise some patience.

**Comment by endrift** [comment]
I'm curious what the PR is waiting on. There doesn't seem to be any reason not to merge it, afaict.

**Comment by WinterMute** [comment]
> I'm curious what the PR is waiting on. There doesn't seem to be any reason not to merge it, afaict.

When the temporary ban gets lifted then it would be appropriate to have an example illustrating the use of the wrapper. Demanding that PRs get merged immediately is disrespectful, we have many other things that require attention and rude users will be deprioritised.

**Comment by gamemasterplc** [comment]
I've written some code using the wrapper at https://github.com/gamemasterplc/warpdemo/blob/master/source/warpdemo.c

**Comment by endrift** [comment]
Maybe that should be added to the examples repository after this is merged.

**Comment by gamemasterplc** [comment]
It can as long as I am credited for creating it @endrift

**Comment by DacoTaco** [comment]
Merged. next time @gamemasterplc, don't be so rude to people if you can. this is what you get then

**Pull request merged**